blocks/search-results/search-results.js

```diff
@@ -85,7 +88,7 @@ async function searchPages(placeholders, term, page) {
     let childSpan;
 
     if (parentPath) {
```

Comment:
This logic was mainly for the sunstar engineering search results.
is the same applicable on sunstar also ?

Reply:
Yes, looking at the search results, its applicable here as well.